features/step_definitions/tweet_steps.rb

```diff
@@ -0,0 +1,8 @@
+When(/^I fill in a tweet$/) do
+  @tweet = 'This is a Tweet'
```

Comment:
I think I would have put this in the Scenario rather than in the step so it's a little more user-centric. See my comment on the tweet feature file.